test/integration/configuration.test.js

```diff
@@ -24,7 +24,7 @@ describe(Support.getTestDialectTeaser('Configuration'), function() {
       } else if (dialect === 'mssql' || dialect === 'postgres') {
         return expect(seq.query('select 1 as hello')).to.eventually.be.rejectedWith([seq.HostNotReachableError, seq.InvalidConnectionError]);
       } else {
```

Comment:
Looks like the else would no longer be needed? could just be folded with mssql/dialect? Or rather the else if mssql/postgres is not needed i suppose.